src/backend/TrafficCourts/Citizen.Service/Models/Dispute/NoticeOfDispute.cs

```diff
@@ -156,13 +176,15 @@ public class NoticeOfDispute
         /// The description of the issue with OCR ticket if the citizen has detected any.
         /// </summary>
         [JsonPropertyName("disputant_ocr_issues_description")]
+        [RequiredIf("DisputantDetectedOcrIssues", true)]
```

Comment:
This rule was not specified in the Jira ticket that I refer for validations. However, I thought that a description for the OCR issue should be required if the disputant selected the checkbox for OCR issues detected because there would be no value of checking the box without knowing what is actually scanned wrong. Please correct me if this is incorrect. If this is correct, then the front-end rule should be updated to match this as well.

I will close this PR and create a separate PR since I'm changing the approach to validate fields so it won't be directly going to main, it will be merged into develop first.

Reply:
Okay, so you're going to close this PR?

Reply:
Yes, I will after I create the new PR.